packages/babel-preset-env/data/built-in-features.js

```diff
@@ -79,7 +79,33 @@ const es2015 = {
   },
 
   "es6.map": "Map",
-  "es6.set": "Set",
+
+  // This is explicit due to prevent the stage-1 Set proposals under the
+  // category "Set methods" from being included.
```

Comment:
Interesting. I didn't think that it could cause any problems.

Reply:
Yeah... currently, the the mapping name (in this case "Set") gets matched via `indexOf`, so the es6 compat-table data `Set` and the esnext `Set methods` both match.

We can improve this for sure, but I wanted to keep the [upgrade PR](https://github.com/babel/babel/pull/7808) small.